tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -75,5 +75,26 @@ int main() {
     vformat_to(back_insert_iterator(output_string), locale::classic(), "}}x", make_format_args());
     assert(output_string == "}x");
 
+    output_string.clear();
+    vformat_to(back_insert_iterator(output_string), locale::classic(), "{}", make_format_args((const char*) "f"));

```

Comment:
We conventionally avoid C-style casts everywhere (because they can do so many things, and are impossible to search for). Occurs below.

Reply:
I would also say, that we should fix the bug and not try to work around it

Reply:
We're going to merge and fix the bug in a PR, just because that unblocks Elnar and I have to get a foot X-Ray before fixing the overload set